packages/camera/camera_avfoundation/example/ios/RunnerTests/CameraCaptureSessionQueueRaceConditionTests.m

```diff
@@ -8,13 +8,33 @@
 #endif
 @import XCTest;
 
+#import "MockCameraDeviceDiscoverer.h"
+#import "MockCaptureDevice.h"
+#import "MockCaptureSession.h"
+#import "MockFlutterBinaryMessenger.h"
+#import "MockFlutterTextureRegistry.h"
+#import "MockGlobalEventApi.h"
+
 @interface CameraCaptureSessionQueueRaceConditionTests : XCTestCase
 @end
 
 @implementation CameraCaptureSessionQueueRaceConditionTests
 
 - (void)testFixForCaptureSessionQueueNullPointerCrashDueToRaceCondition {
-  CameraPlugin *camera = [[CameraPlugin alloc] initWithRegistry:nil messenger:nil];
+  MockCaptureDevice *captureDevice = [[MockCaptureDevice alloc] init];
```

Comment:
nit: maybe create a helper `createCameraPlugin` like in other tests, in case we want to add more test functions in this file.